packages/experiments/src/components/Tile/Tile.tsx

```diff
@@ -86,6 +89,8 @@ export class Tile extends BaseComponent<ITileProps, ITileState> {
       isSelected: isSelected,
       isModal: isModal
     };
+
+    this._events = new EventGroup(this);
```

Comment:
this._events needs to be disposed in componentWillUnmount.

Reply:
https://github.com/OfficeDev/office-ui-fabric-react/pull/9810/commits/c72a0014f9686342bd7acabaabe8fecf0c88af71